packages/desktop-client/src/components/common/Block.tsx

```diff
@@ -7,12 +7,12 @@ type BlockProps = HTMLProps<HTMLDivElement> & {
 };
 
 export default function Block(props: BlockProps) {
-  const { style, innerRef, ...restProps } = props;
+  const { className = '', style, innerRef, ...restProps } = props;
   return (
     <div
       {...restProps}
       ref={innerRef}
-      className={`${props.className || ''} ${css(props.style)}`}
+      className={`${className || ''} ${css(style)}`}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏you can now remove the fallback value

```suggestion
      className={`${className} ${css(style)}`}
```